### PR TITLE
DoubleClick Bid Manager API v2 support

### PR DESCRIPTION
**Pull request by ap-artemdubovsky**

### CODE REVIEW

File: dv360proxy/dvApi.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const baseApi = require("./baseApi");
+
+const displayVideo360Api = function () {
+    const dvApi = baseApi.configureApiClient("https://displayvideo.googleapis.com/v1", true);
```
Comment (romanych) [inline dv360proxy/dvApi.js]: I see that DV360 got updated to V2 as well. 
https://developers.google.com/display-video/api/v2-migration-guide
There is no deprecation notice yet, but, would it make sense to update beforehand?

File: dv360proxy/api.js [hunk 2]
```diff
@@ -1,195 +1,66 @@
...
+
+    this.getQueryReports_v2 = async ({queryId, pageToken}) => {
+        return await dbmApiV2.getQueryReports(requestId, queryId, pageToken);
     };
 
     this.ping = async () => {
+        return await ping(async () => await dbmApi.getQueries(requestId));
+    }
+
+    this.ping_v2 = async () => {
```
Comment (romanych) [inline dv360proxy/api.js]: do you think we should have `ping_v2` or we can just migrate `ping` to V2 API? Request and Response is not tied to DB API version, so I don't see a reason to have `ping_v2`

**Review by romanych — CHANGES_REQUESTED** [review]
Great work. Let's migrate DV API to v2 and consider not introducing `ping_v2`

File: dv360proxy/dvApi.js (hunk 1 above)
Comment (ap-artemdubovsky) [reply dv360proxy/dvApi.js]: Done, now ping_v2 uses DV360 API V2

File: dv360proxy/api.js (hunk 2 above)
Comment (ap-artemdubovsky) [reply dv360proxy/api.js]: Done, ping V1 is no more and ping V2 is now just ping

**Pull request merged**